### PR TITLE
Pdmo 80- UI Enhancement

### PR DESCRIPTION
**Pull request by Farhanisto**
## JIRA TICKET NAME:

[PDMO-80 UI Enhancement](https://issues.openmrs.org/browse/PDMO-80)

## SUMMARY:
1:- Use either + icon or the add button to keep consistency
2: similarly use either the pencil icon or the edit button or edit link to keep consistency
3.Manage the spacing between components eg the addresses and the Identifiers components have no space between them, also the space between the name and the demographics component should be increased.

### CODE REVIEW

**Comment by brayoh** [comment]
Looks great 👍

File: app/js/components/identifiers/identifiers.jsx [hunk 1]
```diff
@@ -296,25 +301,15 @@ export default class Identifiers extends React.Component {
                             {this.state.activeCard !== index
                               ? id.identifier
                               : editIdentifiers.identifier || id.identifier}
-                            {<br />}
-                            {this.state.editState === false &&
```
Comment (mkiterian) [inline app/js/components/identifiers/identifiers.jsx]: Why has Edit been removed?

File: app/js/components/identifiers/addIdentifier.jsx [hunk 2]
```diff
@@ -145,12 +146,11 @@ class NewIdentifier extends React.Component {
         {!this.state.isNewIdentifier && !this.props.stateData.editState &&
           <div className="form-group">
             <div className="col-sm-offset-2 col-sm-2 btn-margin">
-              <button
-                type="addButton"
-                name="add"
```
Comment (mkiterian) [inline app/js/components/identifiers/addIdentifier.jsx:150]: Was the name attribute not being used?

Comment (Farhanisto) [reply app/js/components/identifiers/addIdentifier.jsx:150]: Its usage is just to help in identification ....

File: app/css/obsadmin.css [hunk 3]
```diff
@@ -782,6 +784,12 @@ li>a:hover {
  float: right;
 }
 
+.fa-trash-o{
+  float: right;
+  color: red !important;
+}
+
```
Comment (LWanjiru) [inline app/css/obsadmin.css:791]: Minor: Is it necessary to  have 2 empty lines here?

File: app/js/components/demographics.jsx [hunk 4]
```diff
@@ -221,7 +222,13 @@ export default class Demographics extends React.Component {
             <div className='form-group demo'>
               <label className='control-label col-sm-3'> DOB: </label>
               <div className='col-sm-9'>
-                <h5>{this.state.birthdate}</h5>
+                <div className='col-sm-6'>
+                  <h5>{this.state.birthdate}</h5>
+                </div>
+                <div>
+                <FontAwesome className="fa fa-pencil-square-o"
```
Comment (LWanjiru) [inline app/js/components/demographics.jsx]: Consider using a different format here ie. 

>`<FontAwesome`
`     className="fa fa-pencil-square-o"`
`     onClick={this.handleChange}`
>` /> `

Looks neater. Also, do not leave empty spaces at the end of any of the lines.

Comment (LWanjiru) [reply app/js/components/demographics.jsx]: Remember to indent the two middle lines too.

File: app/js/components/identifiers/addIdentifier.jsx [hunk 5]
```diff
@@ -145,12 +146,11 @@ class NewIdentifier extends React.Component {
...
-                onClick={this.handleAddNew}
-                className="btn btn-success form-control"
-              >Add</button>
+              <div >
+                    <FontAwesome className="fa fa-plus-square-o add-btn"
```
Comment (LWanjiru) [inline app/js/components/identifiers/addIdentifier.jsx]: Implement same format recommended above.

**Review by LWanjiru — COMMENTED** [review]
Overall, good work!

**Comment by Farhanisto** [comment]
@LWanjiru  thanks for the tips

**Pull request merged**